--- dnspython-2.2.1/setup.py.orig	2022-03-07 00:23:06.537358000 +0100
+++ dnspython-2.2.1/setup.py	2022-09-11 16:58:54.746788231 +0200
@@ -9,7 +9,7 @@ package_data = \
 
 extras_require = \
 {'curio': ['curio>=1.2,<2.0', 'sniffio>=1.1,<2.0'],
- 'dnssec': ['cryptography>=2.6,<37.0'],
+ 'dnssec': ['cryptography>=2.6,<38.0'],
  'doh': ['requests-toolbelt>=0.9.1,<0.10.0', 'requests>=2.23.0,<3.0.0'],
  'doh:python_full_version >= "3.6.2"': ['httpx>=0.21.1', 'h2>=4.1.0'],
  'idna': ['idna>=2.1,<4.0'],
